src/main/java/org/sunbird/ratings/service/RatingServiceImpl.java

```diff
@@ -76,28 +74,89 @@ public SBApiResponse getRatings(String activity_Id, String activity_Type, String
     }
 
     @Override
-    public SBApiResponse getRatingSummary(String activity_Id, String activity_Type) {
+    public SBApiResponse getRatingSummary(String activityId, String activityType) {
         SBApiResponse response = new SBApiResponse(Constants.API_RATINGS_SUMMARY);
 
         try {
             validationBody = new ValidationBody();
-            validationBody.setActivity_Id(activity_Id);
-            validationBody.setActivity_Type(activity_Type);
+            validationBody.setActivityId(activityId);
+            validationBody.setActivityType(activityType);
 
             validateRatingsInfo(validationBody, "getSummary");
 
             Map<String, Object> request = new HashMap<>();
-            request.put(Constants.ACTIVITY_ID, activity_Id);
-            request.put(Constants.ACTIVITY_TYPE, activity_Type);
+            request.put(Constants.ACTIVITY_ID, activityId);
+            request.put(Constants.ACTIVITY_TYPE, activityType);
 
             List<Map<String, Object>> existingDataList = cassandraOperation.getRecordsByProperties(Constants.KEYSPACE_SUNBIRD,
                     Constants.TABLE_RATINGS_SUMMARY, request, null);
+
             if (!CollectionUtils.isEmpty(existingDataList)) {
+            Map<String, Object> map = existingDataList.get(0);
```

Comment:
The same is used again below -- declare this above and avoid creating new variable again